content/index.html.haml

```diff
@@ -36,14 +36,56 @@ homepage: true
 
 = partial('downloadbanner.html.haml')
 
-= partial('projectjumbotron.html.haml',
-  :href => 'https://www.cloudbees.com/devops-world',
+-# Carousel Slides
+- slides = []
+
+-# Jenkins SuperPowers
+- slides << {:href => 'https://www.youtube.com/watch?v=qE3tfS7k1VI',
+  :title => 'Jenkins Superpowers',
+  :intro => "At DevOps&nbsp;World&nbsp;-&nbsp;Jenkins&nbsp;World Koksuke Kawaguchi described five superpowers for Jenkins.",
+  :image => {:src => expand_link('images/superpowers/planet.png'), :height => "250px"},
+  :call_to_action => {:text => 'Watch the keynote', :href => 'https://www.youtube.com/watch?v=qE3tfS7k1VI'}}
+
+-# Pipeline
+- slides << { :href => expand_link('doc/pipeline/tour/hello-world'),
+  :title => 'Deliver Continuously with Jenkins Pipline',
+  :intro => "Jenkins Pipeline lets you run you create and maintain your CI/CD pipeline as code, stored and versioned along-side with your project code.",
+  :image => {:src => expand_link('images/superpowers/pipeline-ship.png')},
+  :call_to_action => {:text => 'Get started', :href => '/projects/blueocean'}}
+
+-# Evergreen
+- slides << { :href => expand_link('projects/evergreen'),
+  :title => 'Stay Up-to-date with Jenkins Evergreen',
+  :intro => "Evergreen is an automatically updating rolling distribution of Jenkins. Evergreen provides the end-user with a pre-assembled collection of legos that can be immediately used to implement CI/CD workloads.",
+  :image => {:src => expand_link('images/superpowers/evergreen-ship.png')},
+  :call_to_action => {:text => 'Get Started', :href => '/projects/evergreen'}}
+
+-# CasC
+- slides << { :href => expand_link('projects/casc'),
+  :title => 'Automate Jenkins with Configuration-as-Code',
+  :intro => "Configuration-as-Code automates your Jenkins setup from YAML files, letting you reproduce and/or restore a full environment within minutes.",
+  :image => {:src => expand_link('images/superpowers/casc-ship.png')},
+  :call_to_action => {:text => 'Get Started', :href => '/projects/casc'}}
+
+-# Cloud Native Jenkins
+- slides << { :href => expand_link('sigs/cloud-native'),
+  :title => 'Take Jenkins to the Cloud',
+  :image => {:src => expand_link('images/superpowers/cloud-native-ship.png')},
```

Comment:
There may be some confusion, because Cloud Native Jenkins and Cloud Native SIG are quite different terms if we talk about superpowers. But looks OK anyway